lib/OnyxUtils.d.ts

```diff
@@ -51,6 +51,14 @@ declare function getCallbackToStateMapping(): Record<string, Mapping<string>>;
 /** Getter - returns the default key states. */
 declare function getDefaultKeyStates(): Record<string, OnyxValue<string>>;
 
+/**
+ *
+ * @param connectionID connectionID of the subscriber
+ * @param key key that the subscriber is connected to
+ */
+declare function storeKeyByConnections(connectionID: string, key: OnyxKey): void;
+
+declare function deleteKeyByConnections(connectionID: number): void;
```

Comment:
NAB: Just incase these get refactored or a new function is created which separates them, would you mind adding a function comment for `deleteKeyByConnections` too please?